engine/table/src/test/java/io/deephaven/engine/table/impl/lang/TestQueryLanguageFunctionUtils.java

```diff
@@ -14048,4 +14049,686 @@ public static void test_bigdecimal_comps() {
         TestCase.assertTrue(greaterEquals(fv11, bv10));
         TestCase.assertTrue(greaterEquals(dv11, bv10));
     }
+
+    public static void sameBiOrBd(final Object expected, final Object result) {
+        if (expected instanceof BigDecimal) {
+            TestCase.assertTrue(result instanceof BigDecimal);
+            final BigDecimal exBd = (BigDecimal) expected;
+            final BigDecimal reBd = (BigDecimal) result;
+            assertEquals(0, exBd.compareTo(reBd));
+            return;
+        }
+
+        TestCase.assertTrue(result instanceof BigInteger);
+        final BigInteger exBi = (BigInteger) expected;
+        TestCase.assertTrue(result instanceof BigInteger);
+        final BigInteger reBi = (BigInteger) result;
+        assertEquals(0, exBi.compareTo(reBi));
+    }
+
+    public static void test_biginteger_plus() {
+        final BigInteger bv1 = BigInteger.valueOf(10);
+        final BigInteger bv2 = BigInteger.valueOf(5);
+        final byte yv2 = (byte) 5;
+        final int iv2 = 5;
+        final short sv2 = (short) 5;
+        final long lv2 = 5;
+        final float fv2 = 5.0F;
+        final double dv2 = 5.0;
+        final BigInteger bie = BigInteger.valueOf(15);
+        final BigDecimal bde = BigDecimal.valueOf(15);
+        final Object[] expectedAndresult = new Object[] {
+                bie, plus(bv1, bv2),
+                bie, plus(bv1, yv2),
+                bie, plus(yv2, bv1),
+                bie, plus(bv1, sv2),
+                bie, plus(sv2, bv1),
+                bie, plus(bv1, iv2),
+                bie, plus(iv2, bv1),
+                bie, plus(lv2, bv1),
+                bie, plus(bv1, lv2),
+                bde, plus(bv1, fv2),
+                bde, plus(fv2, bv1),
+                bde, plus(bv1, dv2),
+                bde, plus(dv2, bv1),
+        };
+        for (int i = 0; i < expectedAndresult.length; i += 2) {
+            final Object expected = expectedAndresult[i];
+            final Object result = expectedAndresult[i + 1];
+            sameBiOrBd(expected, result);
+        }
+    }
+
+    public static void test_biginteger_minus() {
+        final BigInteger bv1Left = BigInteger.valueOf(100);
+        final BigInteger bv1Right = BigInteger.valueOf(10);
+        final BigInteger bie = BigInteger.valueOf(90);
+        final BigDecimal bde = BigDecimal.valueOf(90);
+        final Object[] expectedAndresult = new Object[] {
```

Comment:
You need tests with fractional doubles/floats and decimals.

Reply:
Done